sdk/keyvault/azure-security-keyvault-keys/inc/azure/keyvault/keys/key_client.hpp

```diff
@@ -168,15 +175,14 @@ namespace Azure { namespace Security { namespace KeyVault { namespace Keys {
      *
      * @remark Use \p options to control which page to get. If
      * #GetPropertiesOfKeysSinglePageOptions.NextLink is not set, the operation will get the first
-     * page and it will set the `ContinuationToken` from the #KeyPropertiesSinglePage as the next
+     * page and it will set the `NextPageToken` from the #KeyPropertiesSinglePage as the next
      * page of the response if there is a next page.
      *
      * @param options The #GetPropertiesOfKeysSinglePageOptions object to for setting the operation
      * up.
      * @param context A #Azure::Core::Context controlling the request lifetime.
-     * @return Azure::Response<KeyPropertiesSinglePage>
      */
-    Azure::Response<KeyPropertiesSinglePage> GetPropertiesOfKeysSinglePage(
+    KeyPropertiesSinglePage GetPropertiesOfKeysSinglePage(
```

Comment:
Now that you have pageable all the `..SinglePage` apis should be removed.  SinglePage was only necessary before Pageable was available.

Reply:
Let me do that in a new PR, since that's more like a KeyVault breaking change to keep in another PR. Let's keep this PR for adding Pageble